upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12.yaml.template

```diff
@@ -699,10 +738,37 @@ spec:
           initialDelaySeconds: 60
           periodSeconds: 10
           timeoutSeconds: 3
+{{- if .EtcdManaged }}
+        volumeMounts:
+        - mountPath: /var/lib/etcd-config
+          name: etcd-config-path
+          readOnly: true
+        - mountPath: /var/lib/etcd-secrets
+          name: etcd-secrets
+          readOnly: true
+{{- end }}
       hostNetwork: true
       restartPolicy: Always
       serviceAccount: cilium-operator
       serviceAccountName: cilium-operator
+{{- if .EtcdManaged }}
+      volumes:
+      # To read the etcd config stored in config maps
+      - configMap:
+          defaultMode: 420
+          items:
+          - key: etcd-config
+            path: etcd.config
+          name: cilium-config
+        name: etcd-config-path
+        # To read the k8s etcd secrets in case the user might want to use TLS
```

Comment:
These are the Cilium etcd secrets, not the k8s ones, right?

Reply:
Yes. etcd-operator creates the secret `cilium-etcd-secret`.